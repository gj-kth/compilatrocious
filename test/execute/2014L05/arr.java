class Factorial {
    public static void main(String [] str) {
        int [] a;

        a = new int[10];
        System.out.println(a.length);
        a[0] = 35;
        a[1] = 10;
        System.out.println(a[0]);
        System.out.println(a[1]);
    }
}
