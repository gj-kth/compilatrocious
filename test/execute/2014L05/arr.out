10
35
10
